### ISSUE TITLE
Semi-colon shown at the bottom of the VMs List

### ISSUE THREAD

**Issue by lizsurette** [kubevirt/web-ui#58]
We should remove the semi-colon at the bottom of the VMs List.

![screen shot 2018-10-24 at 3 15 21 pm](https://user-images.githubusercontent.com/208455/47455361-aef71380-d79f-11e8-89a5-ebc7caa944d0.png)

**Comment by mareklibra** [comment]
eagle-eyed Liz :-)

**Comment by jelkosz** [comment]
@mareklibra this has been targeted to 1.3 and it seems to be merged only to master. Reopening

**Comment by mareklibra** [comment]
The issue was closed automatically by github thanks to the `Fixes:` text in PR.
We should probably avoid this feature (otherwise nice, imho). Or do you know about better "procedure"?

It's responsibility of the PR author to backport the patch if needed. So the author should do this along pushing the master-PR. Within the PR review, we should be careful about the issue target as well and **request** separate PR addressing that. 

Since this is generic topic, adding others:
FYI: @pcbailey @suomiy , @rawagner , @vojtechszocs

**Comment by jelkosz** [comment]
> The issue was closed automatically by github thanks to the Fixes: text in PR.
> We should probably avoid this feature (otherwise nice, imho). Or do you know about better "procedure"?

Other option is to do it similarly to manageiq where a separate issue is opened per branch. The patch author only contributes the patch to master and all of the patches are cherry-picked in a "bulk" by the stable branch maintainer solving the conflicts on the way. Honestly I find this approach too risky so just mentioning it in a hope that everyone will disagree :)

> 
> It's responsibility of the PR author to backport the patch if needed. So the author should do this along pushing the master-PR. Within the PR review, we should be careful about the issue target as well and request separate PR addressing that.

this is basically how oVirt does it. It is the responsibility of the patch author to make sure the patch will be backported to all branches and closes the bug only once the patch is successfully merged to the bottom-most branch. I prefer this option but maybe Im just biased doing this for years in oVirt :)

> 
> Since this is generic topic, adding others:
> FYI: @pcbailey @suomiy , @rawagner , @vojtechszocs 
@ohadlevy

**Comment by jelkosz** [comment]
okey, bacport merged: https://github.com/kubevirt/web-ui/pull/70 closing issue

**Comment by mareklibra** [comment]
Fixed by https://github.com/kubevirt/web-ui/pull/61 and later re-introduced by https://github.com/kubevirt/web-ui/pull/83

**Comment by mareklibra** [comment]
I'll fix it again
